Update soap connector documentation (#78)
ballerina-platform/module-ballerina-soap: src/soap/Module.md

soap:Options soapOptions = {
    usernameToken: usernameToken
};
var response = soapClient->sendReceive(body, "urn:mediate", soapOptions);
if (response is soap:SoapResponse) {
    io:println(response);
} else {
    io:println(response.message());
}
```

### Usage Example

```ballerina
import ballerina/io;
import ballerina/soap;
  
public function main () {

    soap:Soap12Client soapClient = new("http://ws.cdyne.com/phoneverify/phoneverify.asmx?wsdl");

    xml body = xml `<quer:CheckPhoneNumber xmlns:quer="http://ws.cdyne.com/PhoneVerify/query"> 
         <quer:PhoneNumber>18006785432</quer:PhoneNumber>
         <quer:LicenseKey>0</quer:LicenseKey>
      </quer:CheckPhoneNumber>`;

    var response = soapClient->sendReceive(body);
    if (response is soap:SoapResponse) {
        io:println(response["payload"]);
    } else {
        io:println(response.message());
    }    
}
```

> You can run this example using the following steps:

Reviewer: ```suggestion
Follow the steps below to run this example.
```

> 1. Save the example in a Ballerina file (e.g., `soapExample.bal`)

Reviewer: ```suggestion
1. Save the example in a Ballerina file (e.g., `soapExample.bal`).
```

> 2. Run the file using the command `ballerina run soapExample.bal`

Reviewer: ```suggestion
2. Execute the `ballerina run soapExample.bal` command to run the file.
```

> 3. You will get a response similar to the following

Reviewer: ```suggestion
You will get a response similar to the following.
```
